qiskit/transpiler/passes/basis/unroller.py

```diff
@@ -61,7 +62,10 @@ def run(self, dag):
                 #  backend reports "measure", for example.
                 continue
             if node.name in self.basis:  # If already a base, ignore.
-                continue
+                if isinstance(node.op, ControlledGate) and node.op._open_ctrl:
```

Comment:
Can you also add this to `UnrollCustomDefinitions`?(around https://github.com/Qiskit/qiskit-terra/blob/8756861/qiskit/transpiler/passes/basis/unroll_custom_definitions.py#L61 )

Reply:
Yes